vale/styles/jargon/Technology.yml

```diff
@@ -0,0 +1,326 @@
+---
+action:
+  name: replace
+extends: substitution
+ignorecase: true
+level: warning
+message: Use '%s' instead of '%s'
+# swap maps tokens in form of bad: good
+swap:
+  # yamllint disable rule:line-length # Disable yamlint rule about line length
+  # Sources
+  # - https://github.com/sapegin/textlint-rule-terminology/blob/master/terms.jsonc
+  # - https://github.com/paradite/frontend-encyclopedia/blob/main/scripts/data.json
+  # - https://github.com/huacnlee/autocorrect/blob/main/.autocorrectrc.template
+  # - many others
+  # yamllint enable rule:line-length # Restore yamlint
+  ACL: ACL
+  ACLs: ACLs
+  ActiveMQ: ActiveMQ
+  Adwords: Adwords
+  AirPods: AirPods
+  AJAX: AJAX
+  Aliyun: Aliyun
+  Alpine: Alpine
+  Altassian: Altassian
+  Angular: Angular
+  Ansible: Ansible
+  Apache: Apache
+  API: API
```

Comment:
These terms seem to be identities. Is there a need to have them in `swap`?

Reply:
https://vale.sh/docs/topics/styles/#substitution

I'm using `ignorecsse: true`

So these rules are there to replace any one using

`my api is rich` and reports to use to write `my API is rich`